pkg/autometrics/main.go

```diff
@@ -1,68 +1,26 @@
 package autometrics
 
 import (
+	"context"
 	"fmt"
 	"log"
 	"time"
-
-	"github.com/prometheus/client_golang/prometheus"
 )
 
 var (
-	FunctionCallsCount      *prometheus.CounterVec
-	FunctionCallsDuration   *prometheus.HistogramVec
-	FunctionCallsConcurrent *prometheus.GaugeVec
-	DefBuckets              = []float64{.005, .01, .025, .05, .1, .25, .5, 1, 2.5, 5, 10}
+	DefBuckets    = []float64{.005, .01, .025, .05, .1, .25, .5, 1, 2.5, 5, 10}
```

Comment:
We should probably standardize these across implementations. I used the OpenTelemetry set, which also include .75, 7.5, etc